### ISSUE TITLE
Unexpected inverted y-axis from `FigureResampler`

### ISSUE THREAD

**Issue by darynwhite** [predict-idlab/plotly-resampler#89]
First, thanks for creating this! Our data can be very oversized for Plotly and this has the potential to save us a lot of development time!

**Background:**
I'm developing a library of Plotly figure functions for our varying datatypes (meteorological and oceanographic) and our rain sensor samples at 1-minute intervals, but our data extend from 10 months to 2+ years.

**Problem:**
It's on the verge of perfect, but I cannot determine why the y-axis is inverting.
Here is the code that generates a standard Plotly `Scattergl` plot that will always crash a browser or notebook:
```python
fig = plotly.graph_objects.Figure()
fig.add_trace(
    plotly.graph_objects.Scattergl(
        x=pt0.RAIN.data.index,
        y=pt0.RAIN.data.loc[:, IDX[1,"1046",'Vol']],
        mode="markers",
        showlegend=True,
        name="Rain Vol",
    )
)
```
![Rain_NotResampled](https://user-images.githubusercontent.com/40304663/176361473-01f0f1e5-9e2c-4cb1-afb3-775cdd02931b.png)

And here is the code that I'm attempting to use to generate a `Resampler` figure:
```python
fig = FigureResampler(plotly.graph_objects.Figure())
fig.add_trace(
    plotly.graph_objects.Scattergl(
        name="Rain Vol", 
        showlegend=True,
        mode="markers",
    ),
    hf_x=pt0.RAIN.data.index, 
    hf_y=pt0.RAIN.data.loc[:, IDX[1,"1046",'Vol']],
    downsampler=EveryNthPoint(interleave_gaps=False),
    max_n_samples=int(pt0.RAIN.data.index.size / 4),
    limit_to_view=True,
)

fig.show_dash(mode="inline")
```
![Rain_Resampler](https://user-images.githubusercontent.com/40304663/176361690-65829451-db77-4f78-94c4-940fa4456e45.png)

**Comment by jvdd** [comment]
Hi @darynwhite,

Sounds like an awesome project!
Thanks for pointing out this issue :)

This is indeed unexpected / weird behavior. Plotly-Resampler should behave exactly the same as plotly.

I believe in your exanple plotly-resampler considers the datatype of hf_y as string (categorical). Does it work as expected when you call `.astype("float32")` on the series that you pass as `hf_y`?. 

Can you give me more details of the datatype of hf_y (and perhaps also hf_x)?
Giving a minimal reproducible example would be even more awesome!

Cheers, Jeroen

**Comment by jonasvdd** [comment]
Is fixed in plotly-resampler `0.7.2.2`

**Comment by darynwhite** [comment]
Well, that did fix it, and you were correct. Despite it being `NaN`s and `float32` data, Pandas had it be of `dtype: object`. 

Your suggestion of calling it with `.astype("float32")` worked before pulling the PyPi update, but the changes from #90 also fixed it.

Thanks! I look forward to integrating this into our workflows!
